README.md

```diff
@@ -1,3 +1,13 @@
 # Chaos-Charts
 This repository hosts the Litmus Chaos Charts.
 
+## Installation Steps for Chart Releases
+
+- To install the chaos experiments from a specific chart for a given release, execute the following commands
+with the desired `<release_version>`, `<chart_name>` & `<namespace>`
+
+```
+tar -zxvf <(curl -sL https://github.com/litmuschaos/chaos-charts/archive/<release_version>.tar.gz)
+
+find chaos-charts-<release_version> -print0 | grep -FzZ '<chart_name>/experiments.yaml' | xargs kubectl apply -n <namespace> -f
```

Comment:
experiments.yaml or experiment.yaml?

Reply:
Give an example with release-version.tar.gz

Reply:
It is `experiments` @shubhamchaudhary . Kept it at parent level. For specific experiments we will need to do grep with experiment name instead of `experiments`. We can mention this also, I suppose as option.

Reply:
We can provide a description for `<chart_name>` It is creating a little confusion that chart-name is name of category or experiment itself). 
ps: I also got confused when reading it the first time

Reply:
cool. added some more context in the readme